### PR TITLE
[Draft] Fix copyright and reference statements

### PR DESCRIPTION
**Pull request by llFreetimell**
Some of copyright and reference links are incorrect.
This commit will fix it.

ONE-vscode-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

**Comment by llFreetimell** [comment]
This PR will be rebased after #620 is merged

**Comment by llFreetimell** [comment]
### For reviewers

I will re-request after prerequisite PRs are all merged :)

File: media/CircleGraph/external/base.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// This file is copied from
+// https://github.com/lutzroeder/netron/blob/ae449ff55642636e6a1eef092eda34ffcba1c684/source/base.js
```
Comment (seanshpark) [inline media/CircleGraph/external/base.js:2]: Adding something like this is strict requirement from our open source division?
When we have to revise to next version then we have to add all these comments for these and that I think is a waste, which I don't like to do.

Comment (llFreetimell) [reply media/CircleGraph/external/base.js:2]: > Adding something like this is strict requirement from our open source division?

Unfortunately, yes it is IIUC.
Actually, I wrote about this as "Add copyright & license statement of netron, origin at the top of the file." in https://github.com/Samsung/ONE-vscode/pull/518#issuecomment-1111790784 :(

**Comment by seanshpark** [comment]
> Unfortunately, yes it is IIUC.
Actually, I wrote about this as "Add copyright & license statement of netron, origin at the top of the file." in https://github.com/Samsung/ONE-vscode/pull/518#issuecomment-1111790784 :(

OK, I think I misuderstood what you wrote.

**Review by jyoungyun — APPROVED** [review]
LGTM

File: media/CircleGraph/external/base.js (hunk 1 above)
Comment (seanshpark) [reply media/CircleGraph/external/base.js:2]: Could you invide the person in charge for this case. Personlly I would like to know his option as you wrote `IIUC` not `I know`

CC @lemmaa

Comment (llFreetimell) [reply media/CircleGraph/external/base.js:2]: I left comment about this at internal secret repo issue(168)

**Comment by lemmaa** [comment]
These files all come from one place, don't they? I don't think it's necessary to specify the origin for every unmodified file. It is enough to create an ORIGIN file in a folder, specify the representative origin of all files, and move and locate the LICENSE of the project as it is. For files to be modified or newly added, we will need to add copyright boilerplate at the top of the file separately.

**Comment by llFreetimell** [comment]
With https://github.com/Samsung/ONE-vscode/pull/621#issuecomment-1126601142, there is nothing to do anymore.
Close!

**Pull request closed without merging**